process/economics/economicsData.go

```diff
@@ -13,13 +13,15 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process"
 )
 
-var _ process.RewardsHandler = (*EconomicsData)(nil)
-var _ process.FeeHandler = (*EconomicsData)(nil)
+var _ process.EconomicsDataHandler = (*data)(nil)
+var _ process.RewardsHandler = (*data)(nil)
+var _ process.FeeHandler = (*data)(nil)
 
+var epsilon = 0.00000001
 var log = logger.GetOrCreate("process/economics")
 
-// EconomicsData will store information about economics
-type EconomicsData struct {
+// data will store information about economics
+type data struct {
```

Comment:
why this rename? economicsData would have been more appropriate.

Reply:
it would conflict with package name

Reply:
ok